util/argParser.js

```diff
@@ -372,6 +372,12 @@ class ArgParser {
         descripe: "If set, write error messages to redis",
         type: "boolean",
         default: false,
+      },
+
+      "failOnFailedSeed": {
+        describe: "If set, crawler will fail with exit code 1 if initial seed fails",
```

Comment:
I guess this would be if any seed fails, in case there are multiple seeds. But that makes sense to make this an option: fail if any seed fails or not.
I suppose one more would be: fail if all seeds fail, but probably should wait until there's a specific use case for that.
The page list now includes additional info including loadStatus for each page.